File: typical90/004/04.py
H, W = map(int, input().split())
A = [list(map(int, input().split())) for _ in range(H)]

B = [0] * H
C = [0] * W
for i in range(H):
	for j in range(W):
		B[i] += A[i][j]
		C[j] += A[i][j]

ans = [[0] * W for _ in range(H)]
for i in range(H):
	for j in range(W):
		ans[i][j] = B[i] + C[j] - A[i][j]

for i in ans:
	print(*i)

File: typical90/004/04.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
	int H, W;
	cin >> H >> W;
	vector<vector<int>> A(H, vector<int>(W));
	for (int i = 0; i < H; i++) {
		for (int j = 0; j < W; j++) {
			cin >> A[i][j];
		}
	}

	vector<int> B(H), C(W);
	for (int i = 0; i < H; i++) {
		for (int j = 0; j < W; j++) {
			B[i] += A[i][j];
			C[j] += A[i][j];
		}
	}

	vector<vector<int>> ans(H, vector<int>(W));
	for (int i = 0; i < H; i++) {
		for (int j = 0; j < W; j++) {
			ans[i][j] = B[i] + C[j] - A[i][j];
		}
	}

	for (int i = 0; i < H; i++) {
		for (int j = 0; j < W; j++) {
			cout << ans[i][j];
			if (j != W - 1) {
				cout << ' ';
			}
		}
		cout << '\n';
	}
}